benchmark/QR_accuracy_stability_analysis.cc

```diff
@@ -15,148 +15,317 @@
 
 template <typename T>
 static void
-test_cond_helper_0(int64_t m, 
+error_check(int64_t m,
+            int64_t n,
+            int64_t k,
+            T norm_A,
+            T* A_dat,
+            T* A_1_dat,
+            T* Q_dat,
+            T* R_dat,
+            T* I_ref_dat
+            ) {
```

Comment:
Please provide short descriptions of these parameters. Please also use ``const`` for pointers to data that will not change.

Reply:
In general I do want us to document benchmarks. But I will grant that documentation for benchmarking code doesn't need to be as thorough as documentation for proper RandLAPACK code.

Reply:
Done. We'll document benchmarks once they stabilize.